modules/60crypt-ssh/dropbear-start.sh

```diff
@@ -4,7 +4,10 @@
 
 # Linux >= 6.2 allows the TIOCSTI ioctl to be disabled by default;
 # console_auth requires it, so re-enable using the provided sysctl
-[ -w /proc/sys/dev/tty/legacy_tiocsti ] && echo 1 > /proc/sys/dev/tty/legacy_tiocsti
+[ -w /proc/sys/dev/tty/legacy_tiocsti ] && read tiocsti < /proc/sys/dev/tty/legacy_tiocsti && [ "$tiocsti" = "0" ] && {
+  echo 1 > /proc/sys/dev/tty/legacy_tiocsti
+  echo 1 > /tmp/enabled_tiocsti
+}
```

Comment:
These nested and braced chained conditionals are hard to read and also seem more complex than necessary. How about (coupled with suggstion for `dropbear-stop.sh` below)
```suggestion
if [ -w /proc/sys/dev/tty/legacy_tiocsti ]; then
  cp /proc/sys/dev/tty/legacy_tiocsti /tmp/legacy_tiocsti.default >/dev/null 2>&1
  echo 1 > /proc/sys/dev/tty/legacy_tiocsti
fi
```